### ISSUE TITLE
gunicorn problem

### ISSUE THREAD

**Issue by oikaka1** [BlueSquare23/web-lgsm#14]
I'm having a problem with Gunicorn where the server doesn't start but doesn't report any error, but by the debug script the server starts normally

**Comment by BlueSquare23** [comment]
Sorry it took me so long to get back to you. What happens if you try to run the gunicorn server start command in your terminal?

For example,

```
gunicorn --access-logfile "web-lgsm.log" --bind="127.0.0.1:12357" --daemon --worker-class gevent 'app:main()'
```
What does that command give you?

**Comment by wennic** [comment]
When running install.sh I got the following error:

user@ubuntuserver:~/git/web-lgsm$ pip3 install virtualenv
error: externally-managed-environment

After 

sudo apt install python3-virtualenv

running install.sh again succeeded

**Comment by BlueSquare23** [comment]
Okay interesting glad to hear you got it working. I think I see the issue. For Ubuntu the install script tries to install virtualenv via pip. But right yeah instead I'll have to experiment with having it just install it via apt. Thanks for reporting!

**Comment by BlueSquare23** [comment]
This has been fixed in `dev-beta-1.6`. Will close this issue once that's merged aka v1.6 is out.

**Comment by BlueSquare23** [comment]
This has been fix in the new [v1.6 release](https://github.com/BlueSquare23/web-lgsm/releases/tag/v1.6)!
